### PR TITLE
Uml 1018 send activate account

### PR DESCRIPTION
**Pull request by williamfalconeruk**
# Purpose

changes email flow s mentioned in ticket UML-1018

## Approach

No longer throws a conflict exception but rewrites the ExpiryTTL - effectively resending the email as it returns.

## Learning

some bits about dynamodb updates.

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/app/src/App/src/Service/User/UserService.php [hunk 1]
```diff
@@ -414,4 +415,33 @@ public function completeChangeEmail(string $resetToken)
 
         $this->usersRepository->changeEmail($userId, $resetToken, $user['NewEmail']);
     }
+
+    /**
+     * @return string
+     * @throws Exception
+     */
+    public function getLinkToken(): string
```
Comment (cooperaj) [inline service-api/app/src/App/src/Service/User/UserService.php:423]: these should probably all be private.

Comment (williamfalconeruk) [reply service-api/app/src/App/src/Service/User/UserService.php:423]: agreed - will fix.

**Comment by williamfalconeruk** [comment]
not ready yet. needs some work around the fact that a user will exist and activated.

**Review by GemTay — APPROVED** [review]

**Review by nickdavis2001 — APPROVED** [review]
lgtm

**Pull request merged**